README.md

```diff
@@ -83,27 +83,38 @@ unary_operator:
 binary_operator:
     AND | OR |
     < | <= | > | >= | = | <> | != 
-``` 
+```
 
-## Installation
+## Usage from source
 
-### From source
+### To run the app
 
-Download the source to your local computer and start with `docker-compose`:
-```shell script
-docker-compose up -d
+```bash
+docker-compose up app
 ```
 
-This will start the Not-So-BigQuery in a container and will expose it's port to your `localhost:8080`.
+This will start Not-So-BigQuery in a container, and expose port 5443 on your host.
 
-After this, you can configure your library to use `localhost:8080` as the `apiEndpoint` to reach Google BigQuery API.
+After this, you can configure your library to use `http://localhost:5443/` as the API endpoint to reach BigQuery.
 
 ### NodeJS
+
 ```javascript
-const bq = new BigQuery({ apiEndpoint: 'localhost:5550' });
+const bq = new BigQuery({ apiEndpoint: 'localhost:5443' });
+```
+
+### Ruby
+
+```ruby
+bigquery = Google::Cloud::Bigquery.new(endpoint: 'http://localhost:5443/')
+```
+
+### To clear the storage
+
+```bash
+docker-compose rm -fs postgres && docker volume rm -f not-so-bigquery_db-data
```

Comment:
I don't believe that's true - [the docs say Compose V2 is still a release candidate](https://docs.docker.com/compose/cli-command/). The `compose` subcommand is not available on my Arch Linux machine with the latest Docker:

```
➜ docker --version          
Docker version 20.10.8, build 3967b7d28e

➜ docker compose
docker: 'compose' is not a docker command.
See 'docker --help'
```

Reply:
That's interesting, I have the same version, but other build (I'm on Mac):
```
➜ docker --version
Docker version 20.10.8, build 3967b7d

➜ docker compose --help
Usage:  docker compose [OPTIONS] COMMAND
[…]
```

Anyhow, since `docker-compose` is still available, I'll than approve this change.